automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/DataUseAwareSearchSpec.scala

```diff
@@ -33,11 +34,11 @@ class DataUseAwareSearchSpec extends FreeSpec with WebBrowserSpec with UserFixtu
         modal.selectCheckbox("poa")
         modal.submit()
 
-        modal.isVisible shouldBe false
+        eventually { modal.isVisible shouldBe false }
```

Comment:
These changes look unrelated to sign-in... but I guess that's okay if they help test reliability. Any guidance for when we should be using `eventually`?

Reply:
unrelated. impulsive.